components/src/core/ThreeLiner/ThreeLiner.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { HTMLAttributes } from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+import clsx from 'clsx';
+
+export type ThreeLinerClasses = {
+    root?: string;
+    title?: string;
+    subtitle?: string;
+    info?: string;
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+            height: '100%',
+            flexDirection: 'column',
+            justifyContent: 'center',
+            transition: (props: ThreeLinerProps): string =>
+                theme.transitions.create(['all'], {
+                    duration: props.animationDuration || theme.transitions.duration.standard,
+                    easing: theme.transitions.easing.easeInOut,
+                }),
+        },
+        title: {
+            fontSize: '1.875rem',
+            transition: (props: ThreeLinerProps): string =>
+                theme.transitions.create(['all'], {
+                    duration: props.animationDuration || theme.transitions.duration.standard,
+                    easing: theme.transitions.easing.easeInOut,
+                }),
+        },
+        subtitle: {
+            fontSize: '1rem',
+            transition: (props: ThreeLinerProps): string =>
+                theme.transitions.create(['all'], {
+                    duration: props.animationDuration || theme.transitions.duration.standard,
+                    easing: theme.transitions.easing.easeInOut,
+                }),
+        },
+        info: {
+            fontSize: '0.875rem',
+            transition: (props: ThreeLinerProps): string =>
+                theme.transitions.create(['all'], {
+                    duration: props.animationDuration || theme.transitions.duration.standard,
+                    easing: theme.transitions.easing.easeInOut,
+                }),
+            fontWeight: 300,
+        },
+    })
+);
+
+export type ThreeLinerProps = HTMLAttributes<HTMLDivElement> & {
+    /**
+     * First Line Content
+     */
+    title?: string | JSX.Element;
```

Comment:
These should be `ReactNode`, which is 'anything that can be rendered' (numbers, boolean, string, JSX Element)

Reply:
These are updated now to be ReactNodes